select by tag name/apps.js

```diff
@@ -0,0 +1,7 @@
+
+const colors = ["#C2272D", "#F8931F", "#FFFF01", "#009245", "#0193D9", "#0C04ED", "#612F90"];
+const listItems = document.getElementsByTagName('li'); 
+
+for (const i = 0; i < listItems.length; i++) {
```

Comment:
we should use `let` for variable i since this variable changes its value for every valid iteration (increase by 1)
```suggestion
for (let i = 0; i < listItems.length; i++) {
```